gdnative/src/macros.rs

```diff
@@ -231,3 +282,118 @@ macro_rules! godot_test {
         )*
     }
 }
+
+
+#[macro_export]
+macro_rules! godot_create_constructor {
```

Comment:
nit: unless you feel strongly about it, I'd prefer `godot_wrap_constructor` (same for the other similar macros).

Reply:
Sure, I will rename them